splunk-otel-android/src/main/java/com/splunk/rum/RumInitializer.java

```diff
@@ -86,7 +88,7 @@ class RumInitializer {
     }
 
     SplunkRum initialize(Looper mainLooper) {
-        VisibleScreenTracker visibleScreenTracker = new VisibleScreenTracker();
+        ExplicitVisibleScreenNameTracker visibleScreenTracker = new ExplicitVisibleScreenNameTracker();
```

Comment:
This is also something that will need to change, based on divergence from the upstream base code. The `VisibleScreenTracker` is also created by the `OpenTelemetryRumBuilder`, so we'd need to figure out how to NOT have two instances in play.

Reply:
Definitely! For now the only usage, ie. https://github.com/open-telemetry/opentelemetry-android/blob/main/android-agent/src/main/java/io/opentelemetry/android/OpenTelemetryRumBuilder.java#L484 is addressed, because we replace their `ScreenAttributesSpanProcessor` with our own.